testing/dart/image_dispose_test.dart

```diff
@@ -126,6 +126,25 @@ void main() {
     expect(frame2.image.clone()..dispose(), isNotNull);
     frame2.image.dispose();
   });
+
+  test('debugDisposed works', () async {
```

Comment:
We should also test the web implementations. Web tests live under `lib/web_ui/test`.

Reply:
Also, canvaskit tests are separate from HTML. `test/canvaskit` vs `test/engine/surface`.

Reply:
Done